applications/source/file-source/pom.xml

```diff
@@ -26,6 +26,12 @@
             <artifactId>file-supplier</artifactId>
             <version>${java-functions.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud.stream.app</groupId>
+            <artifactId>stream-applications-composite-function-support</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This isn't strictly needed unless we write some tests, but I included in some of the sources in anticipation and just in case it causes any issues. It did catch a test failure in one source that did not include the function definition.